editor/configs/emsMatchingPeopleColumns.ts

```diff
@@ -0,0 +1,16 @@
+"use client";
```

Comment:
why do we need  "use client" at the top of this file? I am still fuzzy on what the "use client" directive does and when to use it

Reply:
this is just copy/pasta cruft. we don't need it here 🙏